core/src/main/scala/org/http4s/multipart/MultipartParser.scala

```diff
@@ -68,9 +68,10 @@ object MultipartParser {
             emit(tail)
           case (line, tail) if isEndLine(line) =>
             halt
+          case (ByteVector.empty, tail) => // This case smells funny to me.
```

Comment:
MultipartSpec.decodeMultipartRequestWithContentTypes -- but no other -- test needed this.  Not yet sure why.